src/core/sign.ml

```diff
@@ -29,7 +29,8 @@ type t =
 
 (** [create path] creates an empty signature with module path [path]. *)
 let create : Path.t -> t = fun sign_path ->
-  { sign_path; sign_symbols = ref StrMap.empty; sign_deps = ref PathMap.empty
+  { sign_path; sign_symbols = ref StrMap.empty
+  ; sign_deps = ref (PathMap.singleton (Path.ghost "unif_rule") [])
```

Comment:
use Unif_rule.path instead

Reply:
Ah but I can't because `Unif_rule` is in `sig_state`, or I move create to `sig_state`.

Reply:
Ah right.

Reply:
A solution would be to have a `Sign.create` that creates an *empty* signature, and a `Sig_state.create` that creates a signature with the pervasive dependencies set. Is this OK?

Reply:
Yes, why not. But it seems better in this case to rename Sign.create into Sign.empty.